resolwe_bio/tools/bamtobigwig.sh

```diff
@@ -5,15 +5,36 @@ source /etc/profile.d/resolwe-base.sh
 bam_file="$1"
 number_of_cores="$2"
 bin_size="$3"
+time="$4"
 
 NAME=`basename "${bam_file}" .bam`
 
+#get bam size in bytes
+bam_size=$(stat -c%s "${NAME}.bam")
+#if bam_size is less than 1 GB (1073741824bytes) set it to 1 GB
+if [[ ${bam_size} -lt 1073741824 ]]
+then
+  bam_size=1073741824
+fi
+
+time_deeptools=$(echo print ${time}*${bam_size}/1073741824. | python)
```

Comment:
Rather use the following line::

    time_deeptools=`python -c "import os; print(max(os.path.getsize('${bam_file}') / 1024**3, 1) * ${time_limit})"`

This also removes the upper part, needed for file bam-file size calculation.